VolunteerApp/ClientApp/src/app/app.module.ts

```diff
@@ -1,34 +1,34 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
-import { FormsModule } from '@angular/forms';
+// import { FormsModule } from '@angular/forms';
```

Comment:
Please remove

NOTE:
Code is always better than comments. So always avoid comments where code can be made self explanatory.

Reply:
fixed